schema/omega-services.async.api.yaml

```diff
@@ -1,525 +1,480 @@
-asyncapi: '2.5.0'
-
+asyncapi: 2.6.0
+id: http://service.catalogue.nationalarchives.gov.uk
 info:
   title: Project Omega Services API
-  version: '1.0.0'
-  description: |
-    The Project Omega API allows you to perform Query and CRUD operations against assets in The National Archives Catalogue.
+  version: 0.1.0
+  description: >
+    The Project Omega API allows you to perform Query and CRUD operations
+    against assets in The National Archives Catalogue.
+
     ### Architecture
-    The API and systems connected to it are using an asynchronous messaging architecture. This may seem unconventional for those who are used to working with HTTP APIs which typically provide a Request-Response approach, but it has many advantages when integrating systems:
+
+    This API and the systems connected to it are using an asynchronous messaging
+    architecture. This may seem unconventional for those who are used to working
+    with HTTP APIs which typically provide a synchronous Request-Response
```

Comment:
It is actually called the [Request-Reply pattern](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RequestReply.html). I think it would be nice to always refer to the message that is sent back as the "reply". I think this is what marks async messaging as different to HTTP. In HTTP you always get a response - it may not be what you want but it should be quick! In async messaging you get a reply which I see as a more considered answer to your request and this is why it might take a bit longer. I think of a reply as like a written letter that gives you answers, whereas a response could just be an automated email saying "we'll get back to you".

Reply:
Agreed. I have appended a commit to address this.